unit-tests/py/rspy/devices.py

```diff
@@ -1,24 +1,38 @@
 # License: Apache 2.0. See LICENSE file in root directory.
 # Copyright(c) 2021 Intel Corporation. All Rights Reserved.
 
+from rspy import log
+
+# We need both pyrealsense2 and acroname. We can work without acroname, but
+# without rs no devices at all will be returned.
 try:
-    from rspy import acroname
+    import pyrealsense2 as rs
+    log.d( rs )
+    try:
+        from rspy import acroname
+    except ModuleNotFoundError:
+        # Error should have already been printed
+        # We assume there's no brainstem library, meaning no acroname either
+        acroname = None
 except ModuleNotFoundError:
-    # Error should have already been printed
-    # We assume there's no brainstem library, meaning no acroname either
+    log.w( 'No pyrealsense2 library is available! Running as if no cameras available...' )
+    import sys
+    log.d( 'sys.path=', sys.path )
+    rs = None
     acroname = None
-import pyrealsense2 as rs
+
 import time
-from rspy import log
 
-_device_by_sn = None
+_device_by_sn = dict()
 _context = None
 
 
 def query():
     """
     Start a new LRS context, and collect all devices
     """
+    if not rs:
+        return
```

Comment:
Need global?

Reply:
It's weird with modules... but yes, I'll add, it's better